README.md

```diff
@@ -132,43 +93,30 @@ npm install
      - OperatorMono-LightItalic.otf
      - etc.
 
-4. From the command line, run:
-
-### Windows
-
-```
-build
-```
+4. From the command line, run the following command, which will generate the new font files in the `build` folder:
+    - Windows: `build`
+    - Linux/Mac: `./build.sh`
 
-### Linux/Mac
-
-```
-./build.sh
-```
-
-This will generate the new font files in the `build` folder. You can now install these fonts on your system.
-
-### Docker
+5. ✔ Install the newly created fonts on your system.
 
+## Using Docker:
+> Note: This requires you to build the docker image as it is not hosted on Dockerhub anymore.
 If you use Docker, you can also use the executable Docker image [here](https://github.com/drod3763/kiliman-operator-mono-lig-docker). It handles all the prerequisite installation for you, you just need to:
 
-1. Clone the repo located at [https://github.com/drod3763/kiliman-operator-mono-lig-docker].
-
-2. Copy your _Operator Mono_ OpenType files into the `input` folder.
+Using Docker Compose cli tool: 
+  1. Clone the repo located at [https://github.com/drod3763/kiliman-operator-mono-lig-docker].
 
-3. From the command line run:
+  2. Copy your _Operator Mono_ OpenType files into the `input` folder.
 
-```
-docker-compose run --rm operator-mono-lig
-```
```

Comment:
Personally, I like code blocks like they have on the left as they are easier for copy/pasting.